build-support/bin/copy_to_s3.py

```diff
@@ -65,7 +70,8 @@ def perform_copy(
     src_prefix: str,
     dst_prefix: str,
     path: str,
-    dst_region: str,
+    region: str,
+    acl: str | None = None,
```

Comment:
The logs bucket is entirely private and has ACLs disabled, so we had to not set an ACL in this case, while still setting one for objects in the binaries bucket.

Reply:
We can create a user for you under the pants account!